products/rds-mysql.md

```diff
@@ -0,0 +1,66 @@
+---
+title: RDS mysql
```

Comment:
Maybe _MySQL on Amazon RDS_ or _Amazon RDS for MySQL_ would be more suitable (I found those terms in the documentation, such as https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_MySQL.html) ?

Reply:
ok sounds good I went with "Amazon RDS for MySQL"